app/models/manageiq/providers/ansible_tower/inventory/persister/automation_manager.rb

```diff
@@ -1,19 +1,18 @@
 class ManageIQ::Providers::AnsibleTower::Inventory::Persister::AutomationManager < ManagerRefresh::Inventory::Persister
+  include ManagerRefresh::Inventory::AutomationManager
+
+  has_inventory automation_manager.credentials
```

Comment:
Soo, `has_inventory` will create an instance method, that lazy instantiates an `InventoryCollection`. 

`automation_manager.credentials` just returns the configuration for that `InventoryCollection`

The lambda's are needed for adding stuff to the configuration, that is only available at runtime.

It's a lot like Rails validations.

```
validates :password, confirmation: true, unless: Proc.new { |a| a.password.blank? }
```